### PR TITLE
Remove Redundant modules to improve performance and upgrade some modules

### PR DESCRIPTION
**Pull request by dbaluku**
https://trello.com/c/DgdwSJyd
Xforms module was not deleted as proposed in the ticket because its still functional and needed 

upgraded modules are
-emr api 
-app ui 
-openmrs ui framework
-Xforms
-Address hierarchy
-HTML form entry

### CODE REVIEW

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -210,4 +210,13 @@
 			SELECT patient_id,(SELECT program.program_id FROM program WHERE program.uuid='de5d54ae-c304-11e8-9ad0-529269fb1459') AS program,encounter_datetime AS date_enrolled,location_id,encounter.creator,NOW(),0,UUID() AS uuid FROM encounter INNER JOIN encounter_type ON(encounter_type.encounter_type_id=encounter.encounter_type) WHERE  encounter_type.uuid='8d5b27bc-c2cc-11de-8d13-0010c6dffd0f' GROUP BY patient_id;
 		</sql>
 	</changeSet>
+
+	<changeSet id="ugandaemr-19072018-1002" author="dbaluku">
+		<comment>Setting  modules to be UgandaEMR Fingerprint,FHIR,Data Entry Statistics to be started manually</comment>
+		<sql>
+			UPDATE global_property SET property_value = 'false' WHERE property = 'ugandaemrfingerprint.started';
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml]: @dbaluku use a single SQL statement UPDATE global_property SET property_value = 'false' WHERE property IN ( 'ugandaemrfingerprint.started', second_property, third_property);

File: pom.xml [hunk 2]
```diff
@@ -56,27 +56,24 @@
...
 		<allergyuiVersion>1.6</allergyuiVersion>
 		<appframeworkVersion>2.11.0</appframeworkVersion>
-		<appuiVersion>1.8.0</appuiVersion>
+		<appuiVersion>1.9.0</appuiVersion>
 		<appointmentschedulingVersion>1.7.0</appointmentschedulingVersion>
-		<appointmentschedulinguiVersion>1.5.0</appointmentschedulinguiVersion>
-		<atlasVersion>2.2</atlasVersion>
 		<calculationVersion>1.2</calculationVersion>
-		<chartsearchVersion>2.1.0</chartsearchVersion>
 		<coreappsVersion>1.20.0-SNAPSHOT</coreappsVersion>
```
Comment (ssmusoke) [inline pom.xml]: @dbaluku Upgrade coreapps to the latest snapshot 1.21.0-SNAPSHOT

File: pom.xml [hunk 3]
```diff
@@ -99,10 +96,10 @@
 		<reportingUIVersion>1.6.0</reportingUIVersion>
 		<serializationxstreamVersion>0.2.14</serializationxstreamVersion>
 		<uicommonsVersion>2.6.1</uicommonsVersion>
-		<uiframeworkVersion>3.13.0</uiframeworkVersion>
+		<uiframeworkVersion>3.15.0</uiframeworkVersion>
 		<uilibraryVersion>2.0.6</uilibraryVersion>
 		<ugandaemrfingerprintVersion>1.0.6-SNAPSHOT</ugandaemrfingerprintVersion>
```
Comment (ssmusoke) [inline pom.xml]: @dbaluku For UgandaEMR fingerprint use 1.0.6 which is the latest released version

File: api/src/main/resources/liquibase.xml (hunk 1 above)
Comment (dbaluku) [reply api/src/main/resources/liquibase.xml]: Done

File: pom.xml (hunk 2 above)
Comment (dbaluku) [reply pom.xml]: Done

File: pom.xml (hunk 3 above)
Comment (dbaluku) [reply pom.xml]: Done

**Pull request merged**